### PR TITLE
Fixing build errors in 0.3

### PR DESCRIPTION
**Pull request by floreseken**
This current example code does not compile in 0.3, I fixed that. But the text would probably need some changes too..

### CODE REVIEW

**Comment by guardrex** [comment]
Thanks @floreseken!

@danroth27 A handful of `public` props seem like they would change to `private` and get the attribute ...

Line 114: `public DateTime StartDate`

Line 130: `public int ParentYear`

Line 374: `public bool IsCompleted`

**Comment by floreseken** [comment]
Yes that too, forgot about the private part..

**Comment by guardrex** [comment]
When Dan confirms that those others should be changed (I think so since we marked them `public` and thus implied that they would be settable from an external component), then one of us can square them all away here.

File: docs/components/index.md [hunk 1]
```diff
@@ -137,7 +137,10 @@ Child component:
 <div> ... </div>
 
 @functions {
+    [Parameter]
     public int Year { get; set; }
```
Comment (danroth27) [inline docs/components/index.md]: Should be private

File: docs/components/index.md [hunk 2]
```diff
@@ -137,7 +137,10 @@ Child component:
...
+    
+    [Parameter]
```
Comment (danroth27) [inline docs/components/index.md]: @rynowak Is `[Parameter]` needed here?

**Comment by guardrex** [comment]
... and these others ... `[Parameter]` + `private`, too?

Line 114: public DateTime StartDate
Line 130: public int ParentYear
Line 374: public bool IsCompleted

**Comment by guardrex** [comment]
@floreseken They wired this repo to require the CLA signature. Would you mind reviewing and signing the CLA?

@danroth27 I made the other changes. This LGTM ... u ok with these updates?

**Pull request merged**